.github/renovate.json5

```diff
@@ -1,35 +1,40 @@
 {
-  "schedule": [
-    "before 3am on the first day of the month"
+  schedule: [
```

Comment:
Weird, why is the diff showing this?

Reply:
Sorry I messed this up when trying to merge master into my branch. I want to pass that Lint Commits test. Now things are cleaned up.